language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/tests/helpers/DummyLedgerClient.scala

```diff
@@ -109,12 +109,28 @@ class DummyLedgerClient(
         offset: LedgerOffset,
         parties: util.Set[String]): Flowable[CompletionStreamResponse] =
       commandCompletions
+    // TODO Untested endpoint
```

Comment:
Rather than a `TODO`, perhaps:

```scala
throw new UnsupportedOperationException("untested endpoint; implement when you need it")
```

Reply:
I'm trying to move the clients in parallel and this is a single file that everybody touches, I'll leave it alone for the time being but I will definitely apply this suggestion when I'm putting all the clients together in the final PR for this ticket. Thanks! :+1: